isic/stats/models.py

```diff
@@ -29,10 +29,11 @@ class Meta(TimeStampedModel.Meta):
         constraints = [
             CheckConstraint(
                 name='download_occurred_before_tracking', check=Q(download_time__lt=F('created'))
-            )
+            ),
         ]
 
     download_time = models.DateTimeField()
     ip_address = models.GenericIPAddressField()
+    user_agent = models.CharField(max_length=200, null=True)
```

Comment:
`CharField` should generally just be `blank`, not `null`: https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.null